CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+## 1.15.0 (2019-10-02)
+
+This feature release adds secure support for Git over HTTPS, updates
+`kubectl` and `kustomize`, de-experimental-ises garbage collection,
+namespace scoping, and manifest generation, and does a lot of internal
+rewiring _without_ changing user-visible functions or the public APIs.
+
+### Fixes
+
+- Reinstate `git-secret` support after accidentally breaking it 
+  during a refactor that landed in `1.14.0` [fluxcd/flux#2429][]
+- Fix error handling in `splitConfigFilesAndRawManifestPaths`
+  [fluxcd/flux#2455][]
+
+### Enhancements
+
+- Support secure Git over HTTPS using credentials from environment
+  variables [fluxcd/flux#2470][]
+- Add a flag `--sync-timeout`, for configuring the timeout of sync
+  operations. This is mainly of interest to people making use of the
+  manifest generation feature, or people who are operating
+  exceptionally large Git repositories [fluxcd/flux#2481][]
+- Update `kubectl` to `1.14.7` and `kustomize` to `3.2.0`
+  [fluxcd/flux#2461][]
+- De-experimental-ise garbage collection, namespace scoping, and
+  manifest generation features [fluxcd/flux#2485][]
+- Improve logged warning about unsupported automated resource kinds
+  [fluxcd/flux#2471][]
+
+## Maintenance and documentation
+
+- Build: upgrade Go to `1.13.1` [fluxcd/flux#2482][]
+- Build: ensure generated template modtimes are Unix epoch
```

Comment:
It would be better to describe the problem solved, as well as the solution, here; e.g., "Avoid spurious diffs in generated files by fixing their modtimes"